<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowGPIO</class>
 <widget class="QMainWindow" name="MainWindowGPIO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowGPIO</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="statusTextBrowser"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Pin:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pinLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Pin Value:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="readLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Configure:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="inputButton">
           <property name="text">
            <string>Input</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="outputButton">
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="releaseAllPinsPushButton">
           <property name="text">
            <string>Release All Pins</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Set:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="highButton">
           <property name="text">
            <string>High</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="lowButton">
           <property name="text">
            <string>Low</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="toggleButton">
           <property name="text">
            <string>Toggle</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Pull:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="upButton">
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downButton">
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="triButton">
           <property name="text">
            <string>Tri</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
